src/components/shared/button/index.tsx

```diff
@@ -0,0 +1,15 @@
+import Button, { ButtonProps } from '@mui/material/Button';
+
+interface Props extends ButtonProps {
+  children: React.ReactNode;
+}
+
+const BasicButton = ({ variant = 'contained', children, ...props }: Props) => {
+  return (
+    <Button fullWidth variant={variant} {...props}>
+      {children}
+    </Button>
+  );
+};
```

Comment:
ButtonProps 에 이미 children 이 포함되어 있는 것 같은데, 혹시 이런식으로 작성하면 동작이 안될까요...?
제가 MUI 를 잘 몰라서 혹시 아니라면 죄송합니다ㅠㅠ
```suggestion
const BasicButton = ({
  variant = 'contained',
  children,
  ...props
}: ButtonProps) => {
  return (
    <Button fullWidth variant={variant} {...props}>
      {children}
    </Button>
  );
};
```

Reply:
혜진님이 말하신대로 해도 작동은 잘됩니다 그래서 공통 컴포넌트에서는 필수로 입력해야 되는 것에 대해서 입력하지 않았을 때 경고가 뜨도록 하기 위해 props를 사용했습니다.  굳이 필요 없을까요?

Reply:
아하~ 그런 의도가 있었다면, 저는 노철님이 작성해주셨던 원래 코드가 더 나은 것 같다고 생각합니다~!ㅎㅎ